examples/text/plot_document_classification_20newsgroups.py

```diff
@@ -171,12 +257,10 @@ def benchmark(clf):
     if hasattr(clf, "coef_"):
         print("dimensionality: %d" % clf.coef_.shape[1])
         print("density: %f" % density(clf.coef_))
-
-        if feature_names is not None:
-            print("top 10 keywords per class:")
-            for i, label in enumerate(target_names):
-                top10 = np.argsort(clf.coef_[i])[-10:]
-                print(trim("%s: %s" % (label, " ".join(feature_names[top10]))))
+        print("top 10 keywords per class:")
+        for i, label in enumerate(target_names):
+            top10 = np.argsort(clf.coef_[i])[-10:]
+            print(trim("%s: %s" % (label, " ".join(feature_names[top10]))))
```

Comment:
I think we can remove most the prints of the benchmark function. The long text output of this cell is hard to read.

Reply:
Should I keep the prints of things that were not covered in the demo like dimensionality and density? Or only accuracy and training/testing time?